### PR TITLE
Auto convert using Grafana 11 dashboards

### PR DESCRIPTION
**Pull request by dougszumski**
- Load dashboards into Grafana 11 preview
- View dashboard and export
- Check in changes
  - Don't allow hard coded Prometheus datasources

Needs reviewing / testing.

### CODE REVIEW

**Comment by markgoddard** [comment]
"pluginVersion": "11.0.0-preview", looks a bit weird

**Comment by dougszumski** [comment]
> "pluginVersion": "11.0.0-preview", looks a bit weird

Yeah, 11.0.0 wasn't out then, but it is now (the main advantage for this is that it provides auto-conversion). Dawud did test that they would load back into 10.x.

We can remove `-preview` now.

**Comment by dougszumski** [comment]
Please can you test this @technowhizz ?

I have switched to the non-preview plugin version.

File: etc/kayobe/kolla/config/grafana/dashboards/openstack/node_exporter_full.json
Comment (technowhizz) [inline etc/kayobe/kolla/config/grafana/dashboards/openstack/node_exporter_full.json:1]: The System Processes part of this dashboard is collapsed and so it appears that they haven't been converted

File: etc/kayobe/kolla/config/grafana/dashboards/openstack/rabbitmq.json
Comment (technowhizz) [inline etc/kayobe/kolla/config/grafana/dashboards/openstack/rabbitmq.json:1]: The Queues, Channels and Connections sections have blank panels. The PQL is returning valid data. There seems to be an issue with the panel itself

**Review by technowhizz — CHANGES_REQUESTED** [review]

File: etc/kayobe/kolla/config/grafana/dashboards/openstack/node_exporter_full.json
Comment (technowhizz) [reply etc/kayobe/kolla/config/grafana/dashboards/openstack/node_exporter_full.json:1]: I fixed this in #1069 but it is still the 11.0-preview one

File: etc/kayobe/kolla/config/grafana/dashboards/openstack/rabbitmq.json
Comment (dougszumski) [reply etc/kayobe/kolla/config/grafana/dashboards/openstack/rabbitmq.json:1]: Thanks, will investigate

File: etc/kayobe/kolla/config/grafana/dashboards/openstack/node_exporter_full.json
Comment (dougszumski) [reply etc/kayobe/kolla/config/grafana/dashboards/openstack/node_exporter_full.json:1]: Thanks for that- you can use `sed` to strip off the -preview. I found that worked fine. I will remove node exporter from this, in favour of your PR

Comment (dougszumski) [reply etc/kayobe/kolla/config/grafana/dashboards/openstack/node_exporter_full.json:1]: I've removed this in favour of your PR

File: etc/kayobe/kolla/config/grafana/dashboards/openstack/rabbitmq.json
Comment (dougszumski) [reply etc/kayobe/kolla/config/grafana/dashboards/openstack/rabbitmq.json:1]: It took me ages to find, but the trick is to remove this line:

image

Then it works as expected!

**Review by technowhizz — APPROVED** [review]
LGTM

**Pull request merged**